app/src/org/commcare/dalvik/provider/FixtureDataContentProvider.java

```diff
@@ -49,14 +48,6 @@ public boolean onCreate() {
     public Cursor query(Uri uri, String[] projection, String selection,
             String[] selectionArgs, String sortOrder) {
 
-        IStorageUtilityIndexed<FormInstance> userFixtureStorage;
-
-        try{
-            userFixtureStorage = CommCareApplication._().getUserStorage("fixture", FormInstance.class);
-        } catch(SessionUnavailableException sue){
-            return null;
-        }
-
```

Comment:
Removing this variable, since it was unused.
Pretty sure that expression never raised a SessionUnavailableException, so the catch with 'return null' should be dead code.